src/modules/dashboard/components/Dialogs/CancelExpenditureDialog/CancelExpenditureForm.tsx

```diff
@@ -0,0 +1,263 @@
+      </DialogSection>
+      <DialogSection appearance={{ theme: 'sidePadding' }}>
+        <div className={classNames(styles.row, styles.tokens)}>
+          <FormattedMessage {...MSG.ownersStake} />
+          <div className={styles.value}>
+            <TokenIcon
+              className={styles.tokenIcon}
+              token={token}
+              name={token.name || token.address}
+            />
+            <Numeral unit={getTokenDecimalsWithFallback(0)} value={0.1} />
+            {token.symbol}
+          </div>
+        </div>
+      </DialogSection>
+      <DialogSection>
+        <div className={styles.radioGroup}>
+          <div className={styles.textWrapper}>
+            <FormattedMessage {...MSG.shouldBePenalized} />
+            <IconTooltip
+              icon="question-mark"
+              tooltipText={MSG.effectTooltip}
+              appearance={{ size: 'huge' }}
+              tooltipPopperOptions={{
+                placement: 'top',
+                strategy: 'fixed',
+              }}
+            />
+          </div>
+          <div
+            className={classNames(styles.radioWrapper, {
+              [styles.selected]: values.effect === PenalizeType.Penalize,
+            })}
+          >
+            <Radio
+              name="effect"
+              value={PenalizeType.Penalize}
+              checked={values.effect === PenalizeType.Penalize}
+              elementOnly
+            >
+              <FormattedMessage {...MSG.penalize} />
+            </Radio>
+          </div>
+          <div
+            className={classNames(styles.radioWrapper, {
+              [styles.selected]: values.effect === PenalizeType.Mercy,
+            })}
+          >
+            <Radio
+              name="effect"
+              value={PenalizeType.Mercy}
+              checked={values.effect === PenalizeType.Mercy}
+              elementOnly
+            >
+              <FormattedMessage {...MSG.showMercy} />
+            </Radio>
+          </div>
+        </div>
+        <div
+          className={classNames(styles.message, {
+            [styles.messageWarning]: values.effect === PenalizeType.Penalize,
+          })}
+        >
+          <FormattedMessage
+            {...(values.effect === PenalizeType.Penalize
+              ? MSG.penalizeMessage
+              : MSG.mercyMessage)}
+          />
+        </div>
+      </DialogSection>
+      <DialogSection appearance={{ theme: 'sidePadding' }}>
+        <div className={styles.annotations}>
+          <Annotations
+            label={MSG.textareaLabel}
+            name="annotation"
+            maxLength={90}
+          />
+        </div>
+      </DialogSection>
+      <DialogSection appearance={{ align: 'right', theme: 'footer' }}>
+        <Button
+          appearance={{
+            theme: 'primary',
+            size: 'large',
+          }}
+          style={{
+            width: styles.buttonWidth,
+          }}
+          autoFocus
+          text={MSG.submit}
```

Comment:
Please, use `text={{ id: 'button.submit' }}` instead and delete the button in MSG